mdpow/workflows/dihedrals.py

```diff
@@ -529,45 +678,66 @@ def plot_violins(df, resname, figdir=None, molname=None, width=0.9, solvents=SOL
 
            .. seealso:: :func:`~mdpow.workflows.dihedrals.dihedral_violins`
 
+       *plot_pdf_width*
+           The default value for width of plot output is described in detail under
+           :data:`PLOT_WIDTH_DEFAULT`.
+
        *solvents*
            The default solvents are documented under :data:`SOLVENTS_DEFAULT`.
 
-       :returns:
-
-       *violin plot*
-           returns a :class:`seaborn.FacetGrid` object containing a violin plot of the
-           kernel density estimations (KDE) of the dihedral angle frequencies for each
-           relevant dihedral atom group in the molecule from the current MDPOW project
-
     '''
 
+    assert figdir is not None, "figdir MUST be set, even though it is a kwarg. Will be changed with #244"
```

Comment:
@orbeckst 

Please have a look at the modified test in `test_workflows_base.py` that handles this issue that will be corrected in the project manifest PR.

Reply:
That's ok, I just added a doc string as suggestion.